CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Records breaking changes from major version bumps
 
+## 18.0.0
+
+PR: [#151](https://github.com/alphagov/digitalmarketplace-apiclient/pull/151)
+
+The default request error message for unrecognised/unhandled errors has changed from 'Request failed' to 'Unknown request failure in dmapiclient'. 'Request failed' should be seen only in exceptional circumstances - most exceptions raised from failed requests should now hold the str() and repr() calls of the request error as their message.
+
+Any code that relies on the specific format of the request errors raised by the apiclient will need to be updated.
```

Comment:
This is one of those "as someone who doesn't know the whole of `digitalmarketplace-foo-frontend` like the back of my hand, *how the hell* am I expected to check for this?" lines... not your fault though, just an issue with the whole "not my problem (except it actually is)" philosophy of our architecture.

Reply:
Yeah I mean I understand what you're saying. Really not sure how I'm meant to give more detailed instructions in this case, though...